Gems/ROS2/Registry/sdfassetbuilder_settings.setreg

```diff
@@ -13,7 +13,16 @@
                     "xacro"
                 ],
                 "UseArticulations": true,
-                "URDFPreserveFixedJoint": true
+                "URDFPreserveFixedJoint": true,
+                "AssetResolverSettings":
+                {
+                    "URIPrefixMap": 
```

Comment:
I am not really sure how much we want to keep in `.setreg` files, but shouldn't you add _UseAmentPrefixPath_ and _UseAncestorPaths_ as well?

Reply:
They can both be set this way as well, I just didn't add them because I didn't want to change the default settings. I can add them for visibility and clarity though.

The idea behind the setreg file is that this is a way to globally control the default settings for the importer. You can also create your own global overrides by creating a similar file in your user/Registry folder. 

Eventually, it would be nice to get to a point where we are able to load/save these settings either on a per-file basis or just as arbitrary settings that you can load and save from the Robot Importer wizard, so that way if you keep re-importing the same robot you don't have to keep recreating the same settings.

Reply:
I was thinking mainly about visibility and clarity: it is easier for users to switch known parameters in the file than to look for possible names in the code. On the other hand, all parameters can be changed in the wizard.

Thanks for the explanation.

Reply:
I've added them for the improved visibility.